scheduler/scheduler.go

```diff
@@ -235,7 +246,9 @@ func (sched *Scheduler) runIteration(userScenario []scenario.Action, sessionStat
 				logEntry.Action = nil
 				logEntry.LogError(errors.Wrap(errTimeBuf, "time buffer in-between sequences failed"))
 			}
-			return errors.WithStack(err)
+			if !sched.continueOnErrors {
+				return errors.WithStack(err)
```

Comment:
Maybe log warning since otherwise error won't get reported? Alternative 2, collect all errors in a multierror and return at end.

Reply:
Ah really? I figured it would be added to the logger either way.

Reply:
As far as I can tell, it would only affect the return of the Execute method, logging should still be the same, no? I still want to log the errors as normal.